moment/moment.d.ts

```diff
@@ -276,6 +277,12 @@ declare module moment {
         isSame(b: Date, granularity: string): boolean;
         isSame(b: number[], granularity: string): boolean;
 
+        isBetween(a: Moment, b: Moment, granularity?: string): boolean;
```

Comment:
Thanks for your comments.

I would have used a type alias but these are forbidden in the DefinitelyTyped contribution guide. The other options are ``any`` or enumerating the 25 possibilities.

It seems like calculating ``isBetween()`` on the same type would be the overwhelmingly common use case when using TypeScript. This is because the TypeScript programmer is much more certain what types their variables have.